src/java/picard/illumina/CheckIlluminaDirectory.java

```diff
@@ -30,13 +30,25 @@
  * specified data type.  If NO data type is specified then the default data types used by IlluminaBasecallsToSam are used.
  */
 @CommandLineProgramProperties(
-        usage = "Check that the files to provide the data specified by DATA_TYPES are available, exist, and are reasonably sized for every tile/cycle.  " +
-                "Reasonably sized means non-zero sized for files that exist per tile and equal size for binary files that exist per cycle/per tile. " +
-                "CheckIlluminaDirectory DOES NOT check that the individual records in a file are well-formed.",
-        usageShort = "Asserts the validity of the data in the specified Illumina basecalling data",
+        usage = CheckIlluminaDirectory.USAGE_SUMMARY + CheckIlluminaDirectory.USAGE_DETAILS,
+        usageShort = CheckIlluminaDirectory.USAGE_SUMMARY,
         programGroup = Illumina.class
 )
 public class CheckIlluminaDirectory extends CommandLineProgram {
+    static final String USAGE_SUMMARY = "Asserts the validity of the data in the specified Illumina basecalling (.bcl) data.  ";
```

Comment:
"validity of the data in the specified Illumina basecalling (.bcl) data"? I think you mean directory. Should probably lose the extension too since it's specific to only a subset of the data in said directory.